explorer/data/prelude.carbon

```diff
@@ -141,23 +141,23 @@ impl String as CompareWith(Self) {
 }
 
 interface LessWith(U:! Type) {
-  fn Less[me: Self](other: U) -> Bool;
+  fn Less[me: Self](other: U) -> bool;
```

Comment:
The changes in this file make sense, but seem unrelated to the topic of this PR. Did they get added by accident?

Reply:
Ignore them... This comes from a local merge! Today, I have fixed a bug in the prelude!

Reply:
This has now been fixed in trunk, so you can probably merge with that to make these go away.